release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -265,6 +265,11 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 [id="ocp-4-14-machine-config-operator"]
 === Machine Config Operator
 
+[id="ocp-4-14-additional-prometheus-metrics"]
+==== Additional metrics available in Prometheus
+
+With this release, you can query additional metrics that let you more closely monitor the state of your machines and machine config pools.
```

Comment:
`let you` would be slightly informal and sounds synonymous to `allow` that ISG does not recommend to be used.
A better wording I could think of. You could choose a better one too.

```suggestion
With this release, you can query additional metrics that help in detailed monitoring of the state of your machines and machine config pools.
```

Reply:
updated